cmd/wego/app/remove/cmd.go

```diff
@@ -0,0 +1,97 @@
+package remove
+
+// Provides support for removing an application from wego management.
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/lithammer/dedent"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/weaveworks/weave-gitops/cmd/wego/version"
+	"github.com/weaveworks/weave-gitops/pkg/flux"
+	"github.com/weaveworks/weave-gitops/pkg/git"
+	"github.com/weaveworks/weave-gitops/pkg/kube"
+	"github.com/weaveworks/weave-gitops/pkg/logger"
+	"github.com/weaveworks/weave-gitops/pkg/osys"
+	"github.com/weaveworks/weave-gitops/pkg/runner"
+	"github.com/weaveworks/weave-gitops/pkg/services/app"
+	"github.com/weaveworks/weave-gitops/pkg/utils"
+)
+
+var params app.RemoveParams
+
+var Cmd = &cobra.Command{
+	Use:   "remove [--auto-merge <true|false>] [--private-key <keyfile>] <app name>",
```

Comment:
I thought auto merge had to always be set to true here? If so we shouldnt show the option.